geode-pulse/build.gradle

```diff
@@ -50,7 +50,9 @@ dependencies {
   // Needed to fully use log4j instead of commons-logging.
   implementation('org.apache.logging.log4j:log4j-jcl')
   implementation('org.apache.logging.log4j:log4j-api')
-//  implementation('org.apache.logging.log4j:log4j-core')
+  implementation('org.apache.logging.log4j:log4j-slf4j-impl') {
```

Comment:
probably it would be good to do this in a separate PR so that we can backport this to support branches.

Reply:
Okay. And probably it should be `runtimeOnly` instead of `implementation`

Reply:
Done: https://github.com/apache/geode/pull/4988

And I rebased this PR on that one.